wdl-lint/src/rules/description_missing.rs

```diff
@@ -37,7 +37,7 @@ fn description_missing(span: Span, parent: SectionParent) -> Diagnostic {
     ))
     .with_rule(ID)
     .with_highlight(span)
-    .with_fix("add a `description` key to this meta section")
+    .with_fix("add a description key to the meta section")
```

Comment:
```suggestion
    .with_fix("add a `description` key to the meta section")
```

We've used the backticks pretty consistently up until this point when referring to code literals, so I feel we should continue to do that unless we make the decision to remove them everywhere (I would not be in favor of that).